src/PayU/Alu/Response.php

```diff
@@ -10,11 +10,13 @@ class Response
 {
     /**
      * @var string
+     * merchantPaymentReference in apiV4
      */
     private $refno;
 
     /**
      * @var string
+     * not set in alu Payments response
```

Comment:
What does it mean "not set in alu Payments response"?

Reply:
Oh, don't bother with that, those comments were meant to help us mapping the fields between ALU v3 and ALU v4. I've removed them.